alert.css

```diff
@@ -0,0 +1,134 @@
+
+  body{
```

Comment:
you should not directly merge into the master branch. create a dev branch from the master and from there merge all your code into dev.